bedrock/firefox/templates/firefox/privacy/index.html

```diff
@@ -87,11 +87,12 @@ <h3 class="privacy-promise-sub-title">{{ ftl('firefox-privacy-hub-youll-always-k
   </div>
 
   <div class="privacy-promise-learn-more">
-    <ul class="mzp-l-content mzp-l-card-half">
-      {% call picto_card(
+      <ul class="mzp-l-content mzp-l-columns mzp-t-columns-two mzp-t-picto-center">
+      {% call picto(
         title=ftl('firefox-privacy-hub-why-trust-firefox'),
-        class='trust',
-        custom_desc=True
+        body=True,
+        image_url='img/icons/mountain-purple.svg',
+        image_width=150
```

Comment:
Since this is in a list it needs `base_el='li',` param, otherwise the macro defaults to a `div`